drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/main/java/org/drools/workbench/screens/testscenario/client/page/configuration/KiePage.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.testscenario.client.page.configuration;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.drools.workbench.models.testscenarios.shared.Scenario;
+import org.drools.workbench.screens.testscenario.client.ScenarioParentWidget;
+import org.drools.workbench.screens.testscenario.client.resources.i18n.TestScenarioConstants;
+import org.jboss.errai.common.client.ui.ElementWrapperWidget;
+import org.uberfire.backend.vfs.Path;
+import org.uberfire.client.mvp.UberElemental;
+import org.uberfire.client.views.pfly.multipage.PageImpl;
+
+@Dependent
+public class KiePage extends PageImpl {
```

Comment:
Warning over naming this ```KiePage``` and having it in ```drools-wb```.

It looks to be specific to Test Scenarios and hence I'd recommend changing the name accordingly.

```KiePage``` implies, to me at least, that it's a generic re-usable page for all KIE related UI work.

Reply:
Sounds reasonable, I will do.